### DISCUSSION TITLE
Multiple issues PrestaShop

### DISCUSSION THREAD

**Discussion by Old-Paul** [PrestaShop/PrestaShop#34798] [Q&A]
I am a first time use of Presta but over 28 years doing web design.

I have encountered multiple problems and when I find that others have reported same problems in the "official" forum I see many pleas for help remain unanswered for over a year so don't think anyone uses or monitors that site.

Issues:

I get error on check out no carriers assigned for address.  I have US Post office as carrier and US address in cart?  Cannot seem to find how to fix that so checkout does not function.

I cannot use IMPORT because it keeps insisting product_id must be unique which they are. If I try to let PS assign that fails with same errors. Tried the "force use" and same errors. This unable to import products.

Has anyone ever successfully used IMPORT or fixed the problem with no carriers assigned?

**Comment by kpodemski** [comment]
Hello @Old-Paul 

> I have encountered multiple problems and when I find that others have reported same problems in the "official" forum I see many pleas for help remain unanswered for over a year so don't think anyone uses or monitors that site.

A forum is a place where community exchanges about the issues, etc., if you search for support from the company you can see [here](https://addons.prestashop.com/en/388-support).

 > I get error on check out no carriers assigned for address. I have US Post office as carrier and US address in cart? Cannot seem to find how to fix that so checkout does not function.

It's probably some misconfiguration. Probably carriers you set up have restrictions that make it impossible for them to be available during the checkout. You can check the user documentation here: https://docs.prestashop-project.org/v.8-documentation/user-guide/improving-shop/managing-shipping/carriers - maybe there will be some information helpful for your problem.

> I cannot use IMPORT because it keeps insisting product_id must be unique which they are. If I try to let PS assign that fails with same errors. Tried the "force use" and same errors. This unable to import products.

Are you sure you assigned columns from .csv to correct fields before import?
